### PR TITLE
3.3.0 version bumps

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

**Review by kacf — CHANGES_REQUESTED** [review]

File: 05.System-updates-Yocto-Project/05.Customize-Mender/01.Delta-update-support/docs.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ command:
 <!--AUTOVERSION: "mender-binary-delta/%/mender-binary-delta-%.tar"/mender-binary-delta-->
 ```bash
 HOSTED_MENDER_EMAIL="myusername@example.com"
-curl -u $HOSTED_MENDER_EMAIL -O https://downloads.customer.mender.io/content/hosted/mender-binary-delta/1.3.0/mender-binary-delta-1.3.0.tar.xz
+curl -u $HOSTED_MENDER_EMAIL -O https://downloads.customer.mender.io/content/hosted/mender-binary-delta/1.3.1/mender-binary-delta-1.3.1.tar.xz
```
[the remark this answers is not in the document]
Comment (oleorhagen) [reply 05.System-updates-Yocto-Project/05.Customize-Mender/01.Delta-update-support/docs.md:29]: Good catch!

[the remark this answers is not in the document]
Comment (oleorhagen) [reply 05.System-updates-Yocto-Project/05.Customize-Mender/01.Delta-update-support/docs.md]: Also debumped the `connect` version.

File: 08.Server-integration/03.Mutual-TLS-authentication/docs.md [hunk 2]
```diff
@@ -240,7 +240,7 @@ docker run \
   -v $(pwd)/server-cert.pem:/etc/mtls/certs/server/server.crt \
   -v $(pwd)/server-private.key:/etc/mtls/certs/server/server.key \
   -v $(pwd)/ca-cert.pem:/etc/mtls/certs/tenant-ca/tenant.ca.pem \
-  registry.mender.io/mendersoftware/mtls-ambassador:mender-3.2.2
+  registry.mender.io/mendersoftware/mtls-ambassador:mender-origin/staging
```
[the remark this answers is not in the document]
Comment (oleorhagen) [reply 08.Server-integration/03.Mutual-TLS-authentication/docs.md:243]: Finally about time to fix that test failure...

[the remark this answers is not in the document]
Comment (oleorhagen) [reply 08.Server-integration/03.Mutual-TLS-authentication/docs.md]: @kacf see https://github.com/mendersoftware/mender-docs/pull/1776/commits/4ba10cffbeb65af8da07ea078e260d37b50422c8 and https://github.com/mendersoftware/mender-docs/pull/1776/commits/31e8e7d31ebc5bc24a3418ee657635d7d84a43b9

[the remark this answers is not in the document]
Comment (oleorhagen) [reply 08.Server-integration/03.Mutual-TLS-authentication/docs.md]: https://github.com/mendersoftware/mender-docs/pull/1776/commits/31e8e7d31ebc5bc24a3418ee657635d7d84a43b9 will be cherried to `master` too ofc.

**Review by kacf — APPROVED** [review]

**Pull request merged**